Refresh.GameServer/Types/Assets/GameSerializationMethod.cs

```diff
@@ -0,0 +1,12 @@
+namespace Refresh.GameServer.Types.Assets;
+
+public enum GameSerializationMethod : byte
```

Comment:
This needs to be renamed to something shorter. Everywhere it's used causes readability problems.

Probably something like `GameDataFormat` or `GameAssetFormat`

Reply:
i like `GameAssetFormat`, i'll go with that